changelog/22.0/22.0.0/summary.md

```diff
@@ -13,3 +14,22 @@
 These are the RPC changes made in this release - 
 
 1. `GetTransactionInfo` RPC has been added to both `VtctldServer`, and `TabletManagerClient` interface. These RPCs are used to fecilitate the users in reading the state of an unresolved distributed transaction. This can be useful in debugging what went wrong and how to fix the problem.
+
+### <a id="vtorc-config-file-changes"/>VTOrc Config File Changes</a>
+
+The configuration file for VTOrc has been updated to now support dynamic fields. The old `--config` parameter has been removed. The alternative is to use the `--config-file` parameter. The configuration can now be provided in both json, yaml or any other format that [viper](https://github.com/spf13/viper) supports.
+
+The following fields can be dynamically changed - 
+1. `InstancePollTime`
+2. `PreventCrossCellFailover`
+3. `SnapshotTopologyInterval`
+4. `ReasonableReplicationLag`
+5. `AuditToBackend`
+6. `AuditToSyslog`
+7. `AuditPurgeDuration`
+8. `WaitReplicasTimeout`
+9. `TolerableReplicationLag`
+10. `TopoInformationRefreshDuration`
+11. `RecoveryPollDuration`
```

Comment:
We should add more flags to dynamic handling, there's no reason not to. One that comes to mind is `--allow-emergency-reparent`.

Reply:
Done